collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -248,6 +248,9 @@ static String safeName(String name) {
       }
       boolean prevCharIsUnderscore = false;
       StringBuilder safeNameBuilder = new StringBuilder(name.length());
+      if (!name.isEmpty() && Character.isDigit(name.charAt(0))) {
+        safeNameBuilder.append("l");
```

Comment:
I don't think this is what you meant

Reply:
@brian-brazil can you elaborate?

Reply:
this puts a non-numeric 'l' in front of labels that start with a number. Should I do something longer/different/more-clear?

Reply:
I'd suggest putting an underscore in front

Reply:
@brian-brazil done.

Reply:
Did you forget to push your change?